_delb/xpath.py

```diff
@@ -36,13 +38,19 @@
 
 def _split(expression: str, separator: str) -> Iterator[str]:
     assert separator not in ('"', "'")
+    cursor = -1
```

Comment:
I'm pretty sure `cursor` must be initialized with `0`.

Reply:
yep, likely. iirc i had changed the comparison operator in the loop and not adjusted this bit.